packages/desktop/lib/electron/processes/main.process.ts

```diff
@@ -198,7 +191,7 @@ export function createMainWindow(): BrowserWindow {
 
         void windows.main.loadURL('http://localhost:8080')
     } else {
-        new AutoUpdateManager()
+        autoUpdateManager = new AutoUpdateManager(windows.main)
```

Comment:
because this is only initialized on the compiled app, i get 

```
Error occurred in handler for 'get-version-details': Error: No handler registered for 'get-version-details'
```
```
TypeError: Cannot read properties of undefined (reading 'getVersionDetails')
```